src/devices/src/virtio/vsock/muxer.rs

```diff
@@ -518,10 +518,11 @@ impl VsockMuxer {
                     addr: Ipv4Addr::new(0, 0, 0, 0),
                     port: 0,
                 };
-                unix.connect(pkt, tsi);
+                let update = unix.connect(pkt, tsi);
                 unix.confirm_connect(pkt);
                 proxy_map.insert(id, Mutex::new(Box::new(unix)));
                 ipc_map.remove(&pkt.dst_port());
```

Comment:
What's the reason to remove the port from the map? That means the connection can only the established once, doesn't it?

Reply:
Right. Do we want to keep it around for multiple connections?

Reply:
I think I would expect that to be the case. Sounds safer for now.

Reply:
Fixed.